libs/ai-endpoints/langchain_nvidia_ai_endpoints/_common.py

```diff
@@ -185,6 +185,13 @@ def available_models(self) -> list[Model]:
                 # so we'll let it through. use of this model will be
                 # accompanied by a warning.
                 model = Model(id=element["id"])
+
+            # add base model for local-nim mode
+            model.base_model = element.get("root")
+
+            if model.base_model and model.id != model.base_model:
+                model.model_type = "lora"
```

Comment:
`model_type: API type (chat, vlm, embedding, ranking, completion)`

lora isn't a different api type.

it looks like you're using this information to always list lora models in available_models. what about simply not filtering when in non-hosted mode?

Reply:
non-hosted gets defined below in the NVIDIAClient and I do not have access to the same. Also I confirmed with the retriever team that model.id != model.base_model this is best practice of identifying the lora variants and keeping it open-ended might cause issues later. 
What do you recommend?

Reply:
If I do not identify it here then in get_available_models method I do not have any other way to filter it as it does not have client.

Other way around is to add lora variants in statics file as lora variants and let them pass through determine_model

Reply:
can you check now?